lib/spack/spack/package_base.py

```diff
@@ -1881,7 +1881,10 @@ def do_install(self, **kwargs):
             verbose (bool): Display verbose build output (by default,
                 suppresses it)
         """
-        PackageInstaller([(self, kwargs)]).install()
+        explicit = kwargs.get("explicit", True)
```

Comment:
Yeah, we should add an `InstallArguments` type in a following PR.

Reply:
Agreed